src/elements/Button/TextButton/TextButton.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {
+	Text, TextStyle, StyleProp,
+} from 'react-native';
+import styles from '../Button.styles';
+import Button, { ButtonProps } from '../Button';
+
+type TextButtonProps = { text: string; textStyle?: StyleProp<TextStyle> } & ButtonProps;
+
+export default ({
```

Comment:
Update props after the Button component has updated the interface with all possible TouchableHighlight properties.